rocAL/rocAL/source/augmentations/color_augmentations/node_blur.cpp

```diff
@@ -26,35 +26,29 @@ THE SOFTWARE.
 
 BlurNode::BlurNode(const std::vector<Tensor *> &inputs, const std::vector<Tensor *> &outputs) :
         Node(inputs, outputs),
-        _sdev(SDEV_RANGE[0], SDEV_RANGE[1])
-{
-}
+        _kernel_size(KERNEL_SIZE_RANGE[0], KERNEL_SIZE_RANGE[1]) { }
 
-void BlurNode::create_node()
-{
+void BlurNode::create_node() {
     if(_node)
         return;
 
-    _sdev.create_array(_graph ,VX_TYPE_UINT32, _batch_size);
-    // _node = vxExtrppNode_BlurbatchPD(_graph->get(), _inputs[0]->handle(), _src_roi_width,_src_roi_height, _outputs[0]->handle(), _sdev.default_array(), _batch_size);
+    _kernel_size.create_array(_graph , VX_TYPE_FLOAT32, _batch_size);
```

Comment:
Kernel size cannot be float, please change this to int/uint

Reply:
Done